test/json/http_proxy.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
@oschreib don't make this test so specific to image-scanner, also an "Hello World!" here would do.